packages/zen-observable-ts/src/zenObservable.ts

```diff
@@ -0,0 +1,69 @@
+declare function require(name: string);
+const _Observable = require('zen-observable');
+
+import { ZenObservable } from './types';
+
+export { ZenObservable };
+
+export type Observer<T> = ZenObservable.Observer<T>;
+export type Subscriber<T> = ZenObservable.Subscriber<T>;
+export type ObservableLike<T> = ZenObservable.ObservableLike<T>;
+
+export default class Observable<T> {
```

Comment:
Since you mentioned `instanceof`, it might be possible to make this class inherit from `_Observable`:
```typescript
export default class Observable<t> extends _Observable {
```

Reply:
Looks like `extend _Observable` causes [ts] 'extends' clause of exported class 'Observable' has or is using private name '_Observable'. I'm not overly worried about this change, since it is an implementation detail that leaks out and `instanceof` will still  work within the `apollo-link`'s exported observable